### PR TITLE
Update masking_mod.F90

### PR DESCRIPTION
**Pull request by AminIlia-NOAA**
Changed minlat in masking_mod.F90 to -78

  Resolves ##83

-->

# Type of change

- Bug fix (fixes something broken)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? YES
- Does this change require a documentation update? NO

-->

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [*] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: src/ocnicepost.fd/masking_mod.F90 [hunk 1]
```diff
@@ -27,7 +27,7 @@ subroutine remap_masks(vfill)
     ! local variables
     integer            :: rc, ncid, varid, n
     character(len=240) :: wgtsfile
-    real               :: minlat = -79.75
+    real               :: minlat = -78.00
```
Comment (DeniseWorthen) [inline src/ocnicepost.fd/masking_mod.F90]: @AminIlia-NOAA Did you verify that this minimum is valid for all OCN resolutions, meaning no spurious interpolated values appear at any rectilinear resolution? 

The last valid ocean grid point is at 

1/4deg 78.525S
1/2deg 77.72S
1deg 77.77S
5deg 75S

Comment (AminIlia-NOAA) [reply src/ocnicepost.fd/masking_mod.F90]: Thank you for our comment @DeniseWorthen . It is very good point. 
I think it is the best to move it to -78.63 so we don't loose any computational points in 1/4 deg. We may still have spurious results in 5 deg resolution, however, I believe 5deg resolution has no application in today's world.

I committed a change for minlat=-78.63.

Comment (AminIlia-NOAA) [reply src/ocnicepost.fd/masking_mod.F90]: @DeniseWorthen Thank you for your comment. 

Yes, this should resolve it. However as I mention the main problem could be source from another thing. We will loose some computational nodes for 1/4 deg however those are outside ocean edge. Please let me know your opinion.

**Comment by DeniseWorthen** [comment]
@aerorahul It wasn't clear to me that it had actually been tested and confirmed to work for all resolutions vs just "it should resolve it". 

Perhaps the latest commit changing the value was the result of that testing?

**Comment by aerorahul** [comment]
> @aerorahul It wasn't clear to me that it had actually been tested and confirmed to work for all resolutions vs just "it should resolve it".
> 
> Perhaps the latest commit changing the value was the result of that testing?

No sure.  If you agree with the change and think it should fix the issue, please approve and we can merge.  It will get tested when we run the tests in the global-workflow.  If an issue exists then, we can follow up with another PR.  
Does that work for you @AminIlia-NOAA @DeniseWorthen?

**Review by aerorahul — APPROVED** [review]
looks good to me.

**Review by DeniseWorthen — APPROVED** [review]

**Pull request merged**